src/lib-components/AlphabeticalBrowseBy.vue

```diff
@@ -0,0 +1,174 @@
+<template lang="html">
+    <div class="alphabetical-browse-by">
+        <h2 class="title">Browse by Last Name</h2>
+        <ul class="alphabet" v-if="!isTablet">
+            <li
+                v-for="letter in alphabet"
+                :key="letter"
+                class="letter"
+                @click="$emit('letterSelected', letter)"
+            >
+                {{ letter }}
+            </li>
+        </ul>
+        <div v-else>
+            <ul class="alphabet">
+                <li
+                    v-for="letter in alphabetFirstHalf"
+                    :key="letter"
+                    class="letter"
+                    @click="$emit('letterSelected', letter)"
+                >
+                    {{ letter }}
+                </li>
+            </ul>
+            <ul class="alphabet">
+                <li
+                    v-for="letter in alphabetSecondHalf"
+                    :key="letter"
+                    class="letter"
+                    @click="$emit('letterSelected', letter)"
+                >
+                    {{ letter }}
+                </li>
+            </ul>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "AlphabeticalBrowseBy",
+    data() {
+        return {
+            alphabetFirstHalf: [
+                "All",
+                "A",
+                "B",
+                "C",
+                "D",
+                "E",
+                "F",
+                "G",
+                "H",
+                "I",
+                "J",
+                "K",
+                "L",
+            ],
+            alphabetSecondHalf: [
+                "M",
+                "N",
+                "O",
+                "P",
+                "Q",
+                "R",
+                "S",
+                "T",
+                "U",
+                "V",
+                "W",
+                "X",
+                "Y",
+                "Z",
+            ],
+            isTablet: false,
+            vw: window.innerWidth,
+            vh: window.innerHeight,
+        }
+    },
+    props: {},
+    computed: {
+        alphabet() {
+            return this.alphabetFirstHalf.concat(this.alphabetSecondHalf)
+        },
+    },
+    created() {
+        window.addEventListener("resize", this.getSize)
```

Comment:
the main library website nuxt repo has vuex to do this, will go over this with you tomorrow.